src/Microsoft.Owin.Security.Authorization.Mvc/ResourceAuthorizeAttribute.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.Owin.Security.Authorization.Mvc
     [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = true, Inherited = true)]
     public class ResourceAuthorizeAttribute : AuthorizeAttribute, IAuthorizeData
     {
-        private const string s_controllerKey = "Owin.AuthorizationController";
+        private const string s_authorizationContextKey = "Owin.AuthorizationContext";
```

Comment:
Changing this key might seem insignificant, but it could break consumers that have coupled to it externally. I'd like to avoid that. Can you please change it back to its previous value?